docs/APISocketOptions.md

```diff
@@ -124,18 +124,26 @@ their characteristics according to the following legend:
 is empty, it's an option derived from UDT. "Version 0.0.0" is the oldest version
 of SRT ever created and put into use.
 
-2. **Binding**: Defines limitation on setting the option. The field is empty if the option
+2. **Restrict**: Defines restrictions on setting the option. The field is empty if the option
 is not settable (see **Dir** column):
 
-    - `pre`: A connecting socket (both as caller and rendezvous) must be set
-prior to calling `srt_connect()` or `srt_bind()` and never changed thereafter.
-A listener socket should be set to "listening" and it will be
-derived by every socket returned by `srt_accept()`.
+	- `pre-bind`: The option cannot be altered on a socket that is already bound (by calling
+`srt_bind()` or any other function doing this, including automatic binding when trying to
+connect, as well as accepted sockets).
+  **NOTE**: The `pre-bind` characteristic applies exclusively to options that:
+  - Change the behavior and functionality of the `srt_bind` call
+  - Concern or set an option on the internally used UDP socket
+  - Concern any kind of resource used by the multiplexer
```

Comment:
Alignment issue

<img width="512" alt="Screenshot 2021-02-04 at 15 04 37" src="https://user-images.githubusercontent.com/12700120/106903714-9f92fe80-66fa-11eb-8183-969104613e9e.png">

Reply:
I moved the "NOTE" sentence up into the preceding paragraph because the extra indentation was applying code format to the text.

Reply:
Yeah, but it doesn't work. I think it would be simply easier if this whole NOTE with the enumeration is moved past the **post** entry as a new paragraph